src/index.ts

```diff
@@ -33,6 +34,7 @@ export default async function(
 
   const highlighter = await shiki.getHighlighter({
     theme: shikiTheme,
+    langs: options.langs,
```

Comment:
If `options.langs` is optional this needs a little check, because `shiki` might crash with `undefined`.

```js
langs: options.langs || []
```

Reply:
Forgot about it, yep.